content/patterns/multicloud-gitops-sgx-hello-world/mcg-sgx-hello-world-getting-started.adoc

```diff
@@ -0,0 +1,325 @@
++
+[source,terminal]
+----
+$ ./gsc sign-image hello-world enclave-key.pem
+----
+. Push secured container image to registry which is available on the cluster:  
+.. Tag secured container image with appropriate tag and push the image to a registry:
++
+[source,terminal]
+----
+$ docker tag gsc-hello-world <DOCKER_REPOSITORY>:<TAG>
+----
+`<DOCKER_REPOSITORY>` is name a docker repository, `<TAG>` is a version of an image
+
+.. Login to Docker Hub:
++
+[source,terminal]
+----
+$ docker login -u=<DOCKER_ID> docker.io 
+----
+`<DOCKER_ID>` is your login to Docker Hub
+
+.. Push the image to registry:  
++
+[source,terminal]
+----
+$ docker push <DOCKER_REPOSITORY>:<TAG>
+----
+. Update the file _charts/all/hello-world-sgx/values.yaml_ with the information about your image in your forked repo with pattern.
+
+[id="deploying-cluster-using-patternsh-file"]
+== Deploying the cluster by using the pattern.sh file
+
+To deploy the cluster by using the _pattern.sh_ file, complete the following steps:
+
+. Login to your cluster by running the following command:
++
+[source,terminal]
+----
+oc login
+----
++
+Optional: Set the `KUBECONFIG` variable for the _kubeconfig_ file path:
++
+[source,terminal]
+----
+export KUBECONFIG=~/<path_to_kubeconfig>
+----
+
+. Deploy the pattern to your cluster. Run the following command:
++
+[source,terminal]
+----
+./pattern.sh make install
+----
+
+Verify that the Operators have been installed.
+
+. To verify, in the *OpenShift Container Platform* web console, navigate to *Operators → Installed Operators* page.
+. Check that the following Operators are installed with  `Succeeded` status (Figure 1): 
+* *Advanced Cluster Management for Kubernetes* 
+* *Intel Device Plugins Operator* 
+* *multicluster engine for Kubernetes* 
+* *Node Feature Discovery Operator* 
+* *Red Hat Openshift GitOps*
+* *Validated Patterns Operator*
++
+.List of Installed Operators for Multicloud GitOps Validated Pattern with SGX
+image::multicloud-gitops-sgx-hello-world/sgx-installed-operators.png[Multicloud GitOps Hub with SGX installed operators,role="related thumb right"]
++
+
+
+== Deploying the cluster by using the Validated Patterns Operator
+
+To install the *Validated Patterns Operator*:
+
+. Log in to the *Openshift Container Platform* web console and select *Operators > OperatorHub*.
+
+. Search for *Validated Patterns Operator*, open it and click *Install*.
++
+.Install Validated Patterns Operator
+image::multicloud-gitops-sgx-hello-world/sgx-validated-pattern-operator.png[Install Validated Patterns Operator,scale=50]
++
+. Choose default settings for the installation mode, namespaces and update strategy and confirm it by clicking *Install*.
+
+. Select *Operators > Installed Operators*.
+
+. Ensure that *Validated Patterns Operator* is listed in the `openshift-operators` project with a status `Succeeded`.
+
+After succeeded installation open *Validated Patterns Operator*, go to *Pattern* tab and click *Create Pattern*.
```

Comment:
After a successful installation open the *Validated Patterns Operator* page. Next, go to *Pattern* tab and click *Create Pattern*.

Reply:
Updated